### PR TITLE
Updated MME conversion factors and cautions 

### PR DESCRIPTION
**Pull request by c-schuler**
Resolution for #14

### CODE REVIEW

File: input/vocabulary/codesystem/codesystem-CDCMMEClinicalConversionFactors.json [hunk 1]
```diff
@@ -6,136 +6,161 @@
...
+    },
+    {
+      "url": "http://hl7.org/fhir/uv/cpg/StructureDefinition/cpg-relatedArtifact",
+      "valueRelatedArtifact": {
+        "type": "citation",
+        "label": "CDC Opioid Prescribing Guideline",
+        "display": "Centers For Disease Control and Prevention (CDC) Guideline for Prescribing Opioids for Chronic Pain — United States, 2016",
+        "citation": "Dowell D, Haegerich TM, Chou R. CDC Guideline for Prescribing Opioids for Chronic Pain — United States, 2016. MMWR Recomm Rep 2016;65(No. RR-1):1–49. DOI: http://dx.doi.org/10.15585/mmwr.rr6501e1",
+        "url": "http://dx.doi.org/10.15585/mmwr.rr6501e1"
+      }
```
Comment (brynrhodes) [inline input/vocabulary/codesystem/codesystem-CDCMMEClinicalConversionFactors.json:86]: Can we update this to be a reference to the 2022 IG?

**Review by brynrhodes — CHANGES_REQUESTED** [review]
This looks great, the only change I'd suggest is to update the reference to the 2022 guideline, instead of the 2016 guideline.

File: input/vocabulary/codesystem/codesystem-CDCMMEClinicalConversionFactors.json (hunk 1 above)
Comment (c-schuler) [reply input/vocabulary/codesystem/codesystem-CDCMMEClinicalConversionFactors.json:86]: Thanks for catching that! Updated

**Review by brynrhodes — APPROVED** [review]
Looks great!

**Comment by gregoryawhite** [comment]
Thanks for the work on the updates!

Looks like there may be an issue with the master branch?

***@***.***
Thanks Again,

Greg

Greg White, M.S., M.A., Security+
Program Analyst
Security Risk Solutions, Inc.
698 Fishermans Bnd., Mt. Pleasant, SC 29464
Cell: (843) 442-0149
Email: ***@***.***
www.SecurityRiskSolutions.com

**Pull request merged**